plugins/linux-pipewire/pipewire.c

```diff
@@ -1227,10 +1226,12 @@ static void select_source(obs_pipewire_data *obs_pw)
 	g_variant_builder_add(&builder, "{sv}", "handle_token",
 			      g_variant_new_string(request_token));
 
-	if (obs_pw->available_cursor_modes & 4)
+	available_cursor_modes = portal_get_available_cursor_modes();
+
+	if (available_cursor_modes & 4)
 		g_variant_builder_add(&builder, "{sv}", "cursor_mode",
 				      g_variant_new_uint32(4));
-	else if ((obs_pw->available_cursor_modes & 2) && obs_pw->cursor.visible)
+	else if ((available_cursor_modes & 2) && obs_pw->cursor.visible)
```

Comment:
Could these constants be made into a `#define` to make it easier to understand what the code does?

Reply:
Added an extra commit enumerating the available values.

Reply:
@notr1ch Is the enum sufficient, or do you prefer defines for a particular reason?

Reply:
The enum should be fine, I couldn't find anything other than numbers upstream so it seems there's no preferred way to represent these values.